### ISSUE TITLE
Form validation errors are not alerted to screen reader users

### ISSUE THREAD

**Issue by mosefer** [NationalSecurityAgency/skills-service#1206]
All form validation errors are not alerted or made accessible to screenreader users nor are they available via tab-navigation. May be able to incorporate vue-announcer to address, other accessibility tickets involve making use of vue-announcer as well.

**Comment by mosefer** [comment]
most input field validation failures appear to be properly alerting to screen readers as soon as the validation error occurs now however:

* Help Url field validation is not alerted to screen reader unless the field is unfocused and refocused
* point increment field validation is not alerted to screen reader on Edit Skill dialog (for value that exceeds maximum) unless the field is unfocused and refocused
* occurrences to completion field validation is not alerted to screen reader on Edit Skill dialog (for value that exceeds maximum) unless the field is unfocused and refocused
* Time window hours field validation is not alerted to screen reader on Edit Skill dialog (for negative value or invalid characters) unless the field is unfocused and refocused
* window's max occurrences is not alerted to the screen reader (for invalid characters) unless the field is unfocused and refocused
* Time window minutes field validation is not alerted to screen reader on Edit Skill dialog (for negative value/invalid characters) unless the field is unfocused and refocused

**Comment by rmmayo** [comment]
Tested all of the above on integration and all fields alerted using the chrome screen reader plugin. Used keyboard to navigate and waited for the screen reader to finish before entering input.
